x-pack/heartbeat/ironbank/rpm-deps.txt

```diff
@@ -0,0 +1,92 @@
+https://ftp.plusline.net/rockylinux/8.6/BaseOS/x86_64/os/Packages/f/fontconfig-2.13.1-4.el8.x86_64.rpm
```

Comment:
Those dependencies are the ones defined in https://repo1.dso.mil/dsop/elastic/beats/heartbeat/-/blob/development/hardening_manifest.yaml (more precisely the ones defined in https://repo1.dso.mil/dsop/elastic/beats/heartbeat/-/merge_requests/43)